app/views/ProjectEdit/ProjectDetailsForm/index.tsx

```diff
@@ -344,9 +339,9 @@ function ProjectDetailsForm(props: Props) {
                     disabled={disabled || pristine}
                     onClick={handleSubmit}
                     variant="primary"
-                    name="saveAndClose"
+                    name="save"
                 >
-                    {_ts('projectEdit', 'saveButtonLabel')}
+                    Save Project
```

Comment:
Why not just have 'Save'? This is the first place where 'Save' is followed by a noun on a button

Reply:
Mockups